CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### State Machine Breaking
 
+* (x/bank) [#12610](https://github.com/cosmos/cosmos-sdk/pull/12610) `MsgMultiSend` now allows only a single input.
```

Comment:
TODO: Add in `UPGRADING.md` the alternative. (For me to remember for next version)